src/buildMathML.js

```diff
@@ -341,6 +341,31 @@ groupTypes.op = function(group, options) {
     return node;
 };
 
+groupTypes.mod = function(group, options) {
+    var inner = [];
+
+    if (group.value.modType === "pod" || group.value.modType === "pmod") {
+        inner.push(new mathMLTree.MathNode(
+            "mo", [makeText("(", group.mode)]));
+    }
+    if (group.value.modType !== "pod") {
+        inner.push(new mathMLTree.MathNode(
+            "mo", [makeText("mod", group.mode)]));
```

Comment:
After looking at http://caniuse.com/#feat=mathml it's probably safe to say that it's not worth supporting due to spotty browser support.  Originally we wanted to use the MathML rendering for a11y reasons so the browser could read the math out loud.  That didn't work that well so we ended up converting the KaTeX parse tree to a string which the browser could read.